src/styles/blocks/header.scss

```diff
@@ -0,0 +1,176 @@
+.header {
+  padding: 52px 120px 64px;
```

Comment:
padding should be for content-wrapper not for header
and in the figma top padding is 36 px not 52 px